modules/gitops-secrets-store-providers.adoc

```diff
@@ -0,0 +1,42 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-secrets-store-providers_{context}"]
+= Secrets store providers
+
+The following secrets store providers are available for use with the Secrets Store CSI Driver Operator:
+
+* AWS Secrets Manager
+* AWS Systems Manager Parameter Store
+* Microsoft Azure Key Vault
+
+As an example, consider that you are using AWS Secrets Manager as your secrets store provider with the SSCSI Driver Operator. The following example shows the directory structure in {gitops-shortname} repository that is ready to use the secrets from AWS Secrets Manager:
```

Comment:
"a" or "the" {gitops-shortname} repository

Reply:
Fixed.